modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4372,23 +4381,34 @@ type PoorWeather implements ProposalClass {
 
 # Constraints (if any) on the observation's position angle.
 type PosAngleConstraint {
-  # The position angle constraint type in use
-  constraint: PosAngleConstraintType!
 
-  # Describes the fixed position angle constraint, if set
-  angle: Angle
+  # The position angle constraint mode in use.  The value will determine whether
+  # the angle is respected or ignored.
+  mode: PosAngleConstraintMode!
+
+  # The fixed position angle.  This will be kept but ignored for UNBOUNDED and
+  # AVERAGE_PARALLACTIC modes.
+  angle: Angle!
```

Comment:
Right, we keep up with the angle always so that switching from one constraint type to another never loses it.  It is only used though for `FIXED`, `ALLOW_FLIP`, and `PARALLACTIC_OVERRIDE`.